tsconfig.json

```diff
@@ -1,9 +1,10 @@
 {
     "compilerOptions": {
-        "target": "esnext",
-        "module": "esnext",
+        "target": "ES2020",
+        "module": "ES2020",
         "lib": ["es2017", "es7", "es6", "dom", "DOM.Iterable"],
```

Comment:
i think you might want to remove these, as far as i understand it these amount to polyfills, which it's best not to include

Reply:
Type polyfills rather than JS polyfills?

Reply:
i think it makes both available? https://www.typescriptlang.org/tsconfig#lib

> TypeScript also includes APIs for newer JS features matching the target you specify; for example the definition for Map is available if target is ES6 or newer.

Reply:
ah no ignore this comment, the stuff it inserts by tartgetting older versions of JS is not related to lib stuff.

does this mean we _must_ include it? or does ts not include what it need by default?

Reply:
I'm a little bit puzzled by the docs tbh. I think they're saying that `lib` will allow you to explicitly set what type definitions are available for different language features/environments. What I'm not clear on is whether this is just a concern for the package author (i.e. these are just used at compile time), or whether these definitions also get included in the declaration files that you generate alongside your package for consumers to use.